Source/DafnyCore/AST/DafnyAst.cs

```diff
@@ -69,6 +69,10 @@ public abstract class INode {
             }
 
             void updateStartEndTokRecursive(INode node) {
+              if (node is null) {
```

Comment:
I purposedly not included the check because it should never be called on null node. Well I guess we can now put a breakpoint in this if for later!

Reply:
Yeah, I suspect one of the elements of `Children` winds up being `null`, even though I agree it should be left out in that case.

I'm not too worried about this, because, if we decided to allow `null` children, this is clearly the right thing to do with them. :)